table-api/src/main/java/io/deephaven/api/AsOfJoinRule.java

```diff
@@ -0,0 +1,5 @@
+package io.deephaven.api;
+
+public enum AsOfJoinRule {
```

Comment:
I'd like to the main javadoc to live on the QST pojo objects, as it can capture the full context without needing to repeat a bunch of javadoc with each overloaded method. There will be one definitive location for all of the table operation documentation, with the table operation methods pointing to them.